src/config/session-management.md

```diff
@@ -40,3 +40,13 @@ service, as waiting for a D-Bus activation can lead to issues.
 There is an alternative D-Bus configuration which takes advantage of elogind for
 features such as seat detection. It requires installing the `dbus-elogind`,
 `dbus-elogind-libs` and `dbus-elogind-x11` packages.
+
+## seatd
+
+[seatd(1)](https://man.voidlinux.org/seatd.1) is a minimal seat management
+daemon and an alternative to elogind.
+
+To use it, install the `seatd` package and enable its service. If you want
+non-root users to access the seatd session, add them to the `seatd` group.
+
+Note that unlike elogind, seatd doesn't set environmental variables for you.
```

Comment:
Not only it doesn't set them, it also doesn't handle the runtime directory at all. I think this should be in the first paragraph: "it provides some elogind features, including allowing Wayland compositors to not be setuid." - or something of the sort.

Reply:
Honestly this seemed obvious to me given it is a `minimal seat daemon`, not an elogind clone... which is why it didn't seem necessary to me to state this explicitly. If you feel there is a need to do so, I would alter the last paragraph to something like `Note that unlike elogind, seatd doesn't do anything besides managing seats.`. But again, this should be kinda obvious given the fact that it is described as "minimal".

Reply:
Never mind, it will be better to state this explicitly.